### PR TITLE
Create integration tests for Consonant rule

### LINKED ISSUE #580: Create integration tests for "Consonant" rule

### PR DESCRIPTION
**Pull request by edsonlimadev**
fix #580

### CODE REVIEW

File: tests/integration/consonant_1.phpt [hunk 1]
```diff
@@ -0,0 +1,11 @@
+--FILE--
+<?php
+require 'vendor/autoload.php';
+
+use Respect\Validation\Validator as v;
+
+v::consonant()->check('bcd');
+v::consonant()->assert('ddd');
+v::not(v::consonant())->check('uou');
+v::not(v::consonant())->assert('aaaaa');
```
Comment (henriquemoody) [inline tests/integration/consonant_1.phpt]: Nice one!

File: tests/integration/consonant_2.phpt [hunk 2]
```diff
@@ -0,0 +1,14 @@
+--FILE--
+<?php
+require 'vendor/autoload.php';
+
+use Respect\Validation\Validator as v;
+use Respect\Validation\Exceptions\ConsonantException;
+
+try {
+	v::consonant()->check('top nos falsetes');
```
Comment (henriquemoody) [inline tests/integration/consonant_2.phpt]: ROFL

Comment (edsonlimadev) [inline tests/integration/consonant_2.phpt]: laughter is good

**Comment by henriquemoody** [comment]
Merged manually by a281ad03d744cf61847a057b4788d10f2c17d8b5 with some coding standard changes.

If you're a Sublime Text user, I recommend you to change your settings to:

```json
	"ensure_newline_at_eof_on_save": true,
	"tab_size": 4,
	"translate_tabs_to_spaces": true,
	"trim_trailing_white_space_on_save": true,
```

That's why I did change your commit, so it's not necessary to contribute, but it's better.

Thanks for contributing! :panda_face:

**Comment by edsonlimadev** [comment]
I actually forgot this adjustment here
Thank you for reminding me

**Pull request closed without merging**